phlop/os.py

```
@@ -48,5 +48,21 @@
+        with open(file, "r") as f:
+            return f.read()
+    except IOError as e:
+        raise RuntimeError(f"Failed to read file {file}: {e}")
+
+
+def read_last_lines_of(file, n=10):
+    try:
+        with open(file, "r") as f:
+            return f.readlines()[:10]
+    except IOError as e:
+        raise RuntimeError(f"Failed to read file {file}: {e}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)